### ISSUE TITLE
Variable is assigned but never read not triggered when variable output from run

### ISSUE THREAD

**Issue by stefandrissen** [Riverside-Software/sonar-openedge#1072]
````
procedure foo:

   define variable c1 as character no-undo. // not flagged
   define variable c2 as character no-undo. // flagged

   run lala ( output c1 ).
   c2 = 'not used':u.

end procedure.
````
The variable is assigned but never read rule only detects c2 and not c1:

![image](https://github.com/Riverside-Software/sonar-openedge/assets/5297464/4b21b105-fb63-4c81-a71d-27713f6dac76)

If function ``lala`` is used intead of procedure lala, then c1 is correctly flagged.

**Comment by gquerret** [comment]
Variables that only used in an output parameter are not reported. That was done following discussions with multiple customers (but not visible on this issue tracker). They had lots of issues being reported because the API that they were using had output parameters that were never used.
During the discussion, I mentioned that a property should control this behavior, but that obviously wasn't implemented. This can easily be done.

**Comment by stefandrissen** [comment]
Was this behavior changed? In our code base is see otherwise unused variables necessary to receive an output decorated with ``{&_proparse_ prolint-nowarn(varusage)}``.

**Comment by gquerret** [comment]
That was added in [2.8.1](https://github.com/Riverside-Software/sonar-openedge/releases/tag/V2.8.1) in July 2020.

**Comment by stefandrissen** [comment]
Thanks, a property really should be introduced. Or, perhaps to reduce the noise of suppression preprocessors, something in variable name, something like ``unused`` similar to the ``uncaught`` escape for the log or throw rule.

**Comment by gquerret** [comment]
I like the idea of the `unused` variable name. A property with a regexp could control this behavior. That would be limited to VariableAssignedButNotRead rule.

**Comment by gquerret** [comment]
Merged in develop.

> Thanks, a property really should be introduced. Or, perhaps to reduce the noise of suppression preprocessors, something in variable name, something like unused similar to the uncaught escape for the log or throw rule.

Done with a regexp
